plugins/fluentd_telemetry_plugin/build/Dockerfile

```diff
@@ -12,6 +13,8 @@ COPY ${SRC_BASE_DIR}/conf/supervisord.conf /etc/supervisor/conf.d/
 COPY ${SRC_BASE_DIR}/scripts/init.sh ${SRC_BASE_DIR}/scripts/deinit.sh /
 COPY ${SRC_BASE_DIR}/conf/tfs_httpd_proxy.conf ${SRC_BASE_DIR}/conf/fluentd_telemetry_plugin.cfg ${SRC_BASE_DIR}/conf/fluentd.conf ${BASE_PATH}/
 
+COPY ${SRC_BASE_DIR}/lib/libfluent-bit.so ${SRC_BASE_DIR}/lib/libraw_msgpack_api.so  ${TELEMETRY_BASE_PATH}/collectx/lib/
```

Comment:
if you copy executables you probably bind to a certain telemetry version. did you consult the Telemetry team?

Reply:
I chose a specific version of the executables because it's fully tested with TFS and changing versions could require TFS adjustments due to dependency changes,